kyuubi-hive-jdbc-shaded/pom.xml

```diff
@@ -44,12 +44,6 @@
             <artifactId>jsr305</artifactId>
             <optional>true</optional>
         </dependency>
-
-        <!-- TODO: mark as optional once provide JAAS-based Kerberos authentication   -->
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-client-api</artifactId>
```

Comment:
If the user needs to use the `delegationToken` authentication method, the user needs to manually add `hadoop-client-api` to the class path, right?

Reply:
yes, either `hadoop-comon` or `hadoop-client-api` is fine